spotify-auth/README.md

```diff
@@ -0,0 +1,55 @@
+# Use Spotify Sign In with Firebase
+
+This sample shows how to authenticate using Spotify Sign-In on Firebase. In this sample we use OAuth 2.0 based authentication to get Spotify user information then create a Firebase Custom Token (using the Spotify user ID).
+
+
+## Setup the sample
+
+Create and setup the Firebase project:
+ 1. Create a Firebase project using the [Firebase Developer Console](https://console.firebase.google.com).
+ 1. Enable Billing on your Firebase the project by switching to the **Blaze** plan, this is currently needed to be able to perform HTTP requests to external services from a Cloud Function.
```

Comment:
Here the developer can switch to the Blaze or the Flame plan. Can you also link to the pricing page: https://firebase.google.com/pricing/ please?